examples/protocols/displayport/displayport-main.stanza

```diff
@@ -0,0 +1,116 @@
+#use-added-syntax(jitx)
+defpackage jsl/examples/protocols/displayport/main:
+  import core
+  import collections
+  import jitx
+  import jitx/commands
+
+
+  import jsl/design/settings
+  import jsl/symbols/SymbolDefn
+  import jsl/symbols/capacitors
+  import jsl/symbols/resistors
+  import jsl/landpatterns/packages
+  import jsl/landpatterns/two-pin/SMT
+
+  import jsl/protocols/displayport
+  import jsl/si/helpers
+
+  import jsl/examples/protocols/displayport/displayport-board
+  import jsl/examples/protocols/displayport/displayport-src
+
+
+public pcb-component block-cap :
```

Comment:
This is duplicated from PCIe - lets setup a common file that can be referenced from all si protocol examples. 

We might also need to add the `value` parameter - I don't see any for this component.

We also might consider making it parametric so that we don't have to create multiple copies for different values used by different application.

Reply:
Ok, I will try to create a common directory and then move the board and selected components to that common directory.